### ISSUE TITLE
The /boot/efi partition is created on the same disk as Windows, even when manually partitioning the disk.

### ISSUE THREAD

**Issue by saber716rus** [linuxmint/ubiquity#63]
The essence of the problem: there are 2 disks - these are SSD and nvme disks. Windows is installed on nvme. Linux is installed on ssd, and /boot/efi is always installed on nvme for some reason, even if it was created on ssd.

Reproducing the problem:
1) create partitions on ssd (/boot/efi, /)

Expected Behavior: Created on ssd

Current behavior: root partition is created on ssd, and /boot/efi is created on nvme

Addition: 
This problem is not reproducible in Ubuntu 23.04

Logs: https://linux-hardware.org/?probe=0dfc0118d2

**Comment by freckhard** [comment]
I just encountered the same phenomenon, it seems that choosing the "device for bootloader installation" at the bottom of the installer does ignore the made selection. Instead the installer just chooses the first efi partition with the flags `boot, esp` it finds.

I have found a simple workaround: remove those flags from the undesired efi partition with gparted, place them only onto the partition you want the bootloader to install to, install mint, and before you reboot, add those two flags back again (to the partition or all partitions you removed them from).

Hope this helps!

**Comment by jpsarda** [comment]
Yes I got the problem too recently, it was documented here with a working turnaround : https://forums.linuxmint.com/viewtopic.php?t=459844

Basically, before proceding to installation, you should remove the `esp` and `boot` flags on the nvme disk efi partition, and it will be ok, the bootloader will be installed on the external usb storage efi partition. It's not that difficult to do, I did it with gparted application. And don't forget to restore esp and boot flags after installation (but even if you forget you can still do it later with live boot key).
